apps/designer/app/designer/features/style-panel/shared/modifier-keys.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+
+export const useModifierKeys = () => {
```

Comment:
Maybe a hit when this is used: used in combination of modifier keys and mouse interactions, e.g. spacing control shift + mouse